vision/module/get_module_depth.py

```diff
@@ -18,8 +18,13 @@ def get_module_depth(depth_image, coordinates):
     Parameters
     ----------
     depth_image: numpy array
```

Comment:
The data type here could be changed to ndarray to be consistent with other documentation.